app/src/main/java/com/alexP/socialnetwork/ui/screens/contacts/ContactsFragment.kt

```diff
@@ -1,36 +1,33 @@
 package com.alexP.socialnetwork.ui.screens.contacts
 
 import android.Manifest
-import android.app.ActivityOptions
-import android.content.Intent
 import android.content.pm.PackageManager
 import android.os.Bundle
-import android.view.LayoutInflater
+import android.view.View
 import androidx.activity.result.contract.ActivityResultContracts
-import androidx.activity.viewModels
 import androidx.core.content.ContextCompat
-import androidx.fragment.app.FragmentTransaction
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
 import androidx.recyclerview.widget.DefaultItemAnimator
 import androidx.recyclerview.widget.ItemTouchHelper
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.alexP.socialnetwork.App
 import com.alexP.socialnetwork.R
-import com.alexP.socialnetwork.databinding.ActivityContactsBinding
-import com.alexP.socialnetwork.ui.screens.addcontact.AddContactFragment
-import com.alexP.socialnetwork.ui.screens.addcontact.MyFragmentFactory
-import com.alexP.socialnetwork.ui.screens.base.BaseActivity
-import com.alexP.socialnetwork.ui.screens.signup.AuthActivity
+import com.alexP.socialnetwork.databinding.FragmentContactsBinding
 import com.alexP.socialnetwork.utils.SpacingItemDecorator
 import com.alexp.contactsprovider.Contact
 import com.google.android.material.snackbar.Snackbar
 
 
-class ContactsActivity : BaseActivity<ActivityContactsBinding>() {
+class ContactsFragment : Fragment(R.layout.fragment_contacts) {
+
+    private lateinit var binding: FragmentContactsBinding
```

Comment:
Incorrect binding declaration in fragment. [Docs](https://developer.android.com/topic/libraries/view-binding#fragments).